conda_smithy/azure_ci_utils.py

```diff
@@ -168,55 +163,59 @@ def register_repo(github_org, repo_name, project_id=AZURE_PROJECT_ID):
             },
         ],
         variable_groups=aclient.get_variable_groups(
-            project=project_id, group_name="anaconda-org"
+            project=project_name, group_name="anaconda-org"
         ),
         type="build",
     )
 
     # clean up existing builds for the same feedstock if present
     existing_definitions: typing.List[
         BuildDefinitionReference
-    ] = bclient.get_definitions(project=project_id, name=repo_name)
+    ] = bclient.get_definitions(project=project_name, name=repo_name)
     if existing_definitions:
         assert len(existing_definitions) == 1
         ed = existing_definitions[0]
         bclient.update_definition(
-            definition=build_definition,
-            definition_id=ed.id,
-            project=ed.project.name,
+            definition=build_definition, definition_id=ed.id, project=ed.project.name
         )
     else:
-        bclient.create_definition(
-            definition=build_definition, project=project_id
-        )
+        bclient.create_definition(definition=build_definition, project=project_name)
 
 
 def build_client() -> BuildClient:
-    return connection.get_client(
-        "vsts.build.v4_1.build_client.BuildClient"
-    )
+    return connection.get_client("vsts.build.v4_1.build_client.BuildClient")
 
 
-def repo_registered(github_org, repo_name, project_id=AZURE_PROJECT_ID):
+def repo_registered(github_org, repo_name, project_name=AZURE_PROJECT_NAME):
     existing_definitions: typing.List[
         BuildDefinitionReference
-    ] = build_client().get_definitions(project=project_id, name=repo_name)
+    ] = build_client().get_definitions(project=project_name, name=repo_name)
 
     return bool(existing_definitions)
 
 
-def enable_reporting(user, repo, project_id=AZURE_PROJECT_ID):
+def enable_reporting(user, repo, project_name=AZURE_PROJECT_NAME):
     bclient = build_client()
-    bdef_header = bclient.get_definitions(project=project_id, name=repo)[0]
+    bdef_header = bclient.get_definitions(project=project_name, name=repo)[0]
     bdef = bclient.get_definition(bdef_header.id, bdef_header.project.name)
-    bdef.repository.properties['reportBuildStatus'] = 'true'
+    bdef.repository.properties["reportBuildStatus"] = "true"
     bclient.update_definition(bdef, bdef.id, bdef.project.name)
 
 
-def get_build_id(user, repo, project_id=AZURE_PROJECT_ID):
+def get_build_id(user, repo, project_name=AZURE_PROJECT_NAME):
+    """Get the neccesary build information to persist in the config file to allow rendering
+    of badges.
+    This is needed by non-conda-forge use cases"""
     bclient = build_client()
-    bdef_header = bclient.get_definitions(
-        project=AZURE_PROJECT_ID, name=repo)[0]
-    bdef: BuildDefinitionReference = bclient.get_definition(
-        bdef_header.id, bdef_header.project.name)
-    return bdef.id
+    bdef_header = bclient.get_definitions(project=AZURE_PROJECT_NAME, name=repo)[0]
```

Comment:
should this be `project_name` instead of `AZURE_PROJECT_NAME`?

Reply:
well spotted